bin/foreman_scap_client

```diff
@@ -46,22 +46,66 @@ ensure_scan_files () {
 }
 
 load_config () {
-  local config_file="/etc/foreman_scap_client/config"
+  local config_file="/etc/foreman_scap_client/config.yaml"
 
-  # Load config file
   if [ -f $config_file ]; then
-    # shellcheck disable=SC1090
-    source $config_file
+    eval "$(awk '
+    /^:server:/ {
+      print "SERVER=" $2;
+    }
+    /^:port:/ {
+      print "PORT=" $2;
+    }
+    /^:timeout:/ {
+      print "TIMEOUT=" $2;
+    }
+    /^:fetch_remote_resources:/ {
+      print "FETCH_REMOTE_RESOURCES=" $2;
+    }
+    /^:http_proxy_server:/ {
+      print "HTTP_PROXY_SERVER=" $2;
+    }
+    /^:http_proxy_port:/ {
+      print "HTTP_PROXY_PORT=" $2;
+    }
+    /^:ca_file:/ {
+      print "CA_FILE=" $2;
+    }
+    /^:host_certificate:/ {
+      print "HOST_CERTIFICATE=" $2;
+    }
+    /^:host_private_key:/ {
+      print "HOST_PRIVATE_KEY=" $2;
+    }
+    /^:ciphers:/ {
+      print "CIPHERS=" $2;
+    }
+
+    /([0-9]+):/ {
```

Comment:
We probably want to match the entire line. This would cause issues when parsing config files which still have the oval bits in it.
```suggestion
    /^([0-9]+):/ {
```